src/index.js

```diff
@@ -1,3 +1,9 @@
 import { Klicky } from "./klicky";
 
 Klicky();
+
+//Click event specifically for clicking the linkedin icon
+import { clickFunction } from "./klicky";
+const element = document.getElementById("linkedin");
```

Comment:
nice job using imports/exports here! Just to keep things clean in this file, can you mount this event listener from within the Klicky function in the `klicky.ts`? We want to use this file just as a means of interacting with the HTML and leave all of our actual code in the `klicky.ts`